dags/common/parsing/xml_extractors.py

```diff
@@ -24,9 +25,12 @@ def __init__(
         self.extra_function = extra_function
         self.logger = get_logger().bind(class_name=type(self).__name__)
 
-    def _get_text_value(self, raw_value):
+    def _get_extracted_value(self, node):
         try:
-            return raw_value.text
+            extracted_value = "".join(
+                [node.text or ""] + [ET.tostring(el).decode("ascii") for el in node]
```

Comment:
this is to get the children's values as well? if yes, maybe a more specific variable naming would be a good option as this trick seems strange at a first glance ;)